### ISSUE TITLE
Input mapping review

### ISSUE THREAD

**Issue by jpmckinney** [open-contracting/infrastructure#34]
I was having trouble commenting in Google Sheets, so I am copying the task here. I'm assuming only the first sheet is relevant; let me know if I should be looking at any others.

> please review the OCDS input mapping I have added to IDS project level mapping table.

> This is based on the mapping you previously documented to OCDS 1.0RC, with updates to reflect changes since then and with some additional fields mapped (e.g. projected and actual cost at completion).

> I've used the language of 'infer' to indicate that, e.g. the buyer of the contract might not necessarily be the project owner, and plan to add some further prose to that effect on the page where the mapping appears.

> We could be more explicit about whether each element should be inferred from just the main construction contract (such as for the project location) or from all related contracts (such as for the total projected value of the project), but I've tried to keep the mapping concise and propose instead providing some guidance in prose on the page where the mapping appears.

**Comment by duncandewhurst** [comment]
I've updated the mapping in the Google Sheet with these changes, with the exception of

**Project scope** where I've updated the mapping to read:

> OCDS: Check ``planning.documents`` for documents with ``.type`` set to `scope` or infer from ``tender.items`` or ``tender.milestones``

Since the top level ``.documents`` array referenced in the output mapping doesn't exist in OCDS and because inferring from `tender.items` or `tender.milestones` is relevant for users who do not have any control over the OCDS data they are using for infrastructure monitoring (in which case finding a document with ``.type`` of 'scope' is unlikely since that code doesn't yet exist in OCDS).

**Project budget approval date** where I've updated the mapping to read:

> OCDS: Check ``planning.documents`` for a document with ``.type`` set to 'budgetApproval' or check ``planning.milestones`` for a milestone with ``.type`` set to 'approval' and infer from ``.dateMet``

For similar reasons

@jpmckinney please also review the OCDS input mapping in the second sheet of the mapping document Google Sheet ("Mapping - Process Level"), the remaining sheets aren't relevant.

**Comment by duncandewhurst** [comment]
Noting that 'scope' has been updated to read 'projectScope' per https://github.com/open-contracting/infrastructure/issues/35

**Comment by duncandewhurst** [comment]
From #46, once the input mapping is added to the docs, links on the following pages will need to be updated:

- [ ] `assess-against-ids.md`
- [ ] `contracts-to-projects.md`

**Comment by duncandewhurst** [comment]
- [ ] TODO from `cost/index.md`, when adding updated mapping tables, format them to better reflect the stages of the IDS.

**Comment by jpmckinney** [comment]
I've left comments in the two sheets.

**Comment by duncandewhurst** [comment]
@jpmckinney and I discussed the input mapping and agreed to move `contractPeriod` to `contractingProcessSummary` and to update the definitions for `contractPeriod` and `contractValue` to make it clear that these should be used for the initial value and duration of the contract, with changes recorded in variations, which appears to be the intent of the definitions in draft guidance note 9.

Those changes are staged in https://github.com/open-contracting/infrastructure/pull/54

Noting that (in-line with the previous definitions) the new definitions repeat some aspects of the OCDS input mapping, once the input mapping is merged into master we should review schema definitions and remove any repetitions.
